### DISCUSSION TITLE
contribution

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#19] [Announcements]
# contribution

提示 本项目处于早期开发阶段 (Early WIP)，如有 bug 请多担待。 本项目使用了 JavaScript 和 TypeScript 实现。 如果你想参与 VanBlog 开发，可以进群哦： VanBlog 开发群 (https://jq.qq.com/?_wv=1027&k=mf2CguM8); 准备知识 整体架构 Vanblog 分为以下几...

https://vanblog.mereith.com/contribution.html

**Comment by rsoish** [comment]
Good.
